.editorconfig

```diff
@@ -0,0 +1,9 @@
+# top-most EditorConfig file
```

Comment:
This file is supported by all editors. With VS Code we would just need the editorconfig plugin. I put this file in all of my apps.